x-pack/test/security_api_integration/tests/user_profiles/bulk_get.ts

```diff
@@ -30,7 +29,9 @@ export default function ({ getService }: FtrProviderContext) {
         )
       );
 
-      // 2. Activate user profiles
+      // 2. Activate user profiles (activation time affects the order in which Elasticsearch returns results, but the
```

Comment:
__note:__ this is what caused tests to fail - the order of the profile activation isn't guaranteed when we activate multiple profiles in parallel, that in its turn affected the order Elasticsearch `Suggest User Profiles` API was returning results.